components/menu/OverflowMenu/style.module.scss

```diff
@@ -6,36 +6,47 @@
 }
 
 .overflow-menu__button {
-  @include circle(26px);
+  @include circle(36px);
   padding: $size-md;
```

Comment:
Legacy size value used

Reply:
Any idea as to what the size should be? From my inspect on zeroheight, it shows as 36px. Link: https://zeroheight.com/73kwjxh8i/p/52140/i/861670